### ISSUE TITLE
label_translation_parameters does not exist

### ISSUE THREAD

**Issue by rbaarsma** [symfony/symfony#31818]
**Symfony version(s) affected**: 4.2.9

**Description**  
I ran `docker-compose exec php composer update "symfony/*" --with-all-dependencies` with symfony version `4.2.*` and got this error:

```
request.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "Variable "label_translation_parameters" does not exist." at /var/www/application/vendor/symfony/twig-bridge/Resources/views/Form/bootstrap_4_layout.html.twig line 227 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): Variable \"label_translation_parameters\" does not exist. at /var/www/application/vendor/symfony/twig-bridge/Resources/views/Form/bootstrap_4_layout.html.twig:227)"}
```

Here's the composer update log
```
  - Updating doctrine/annotations (v1.6.0 => v1.6.1): Loading from cache
  - Updating doctrine/collections (v1.5.0 => v1.6.1): Loading from cache
  - Updating doctrine/persistence (v1.1.0 => 1.1.1): Loading from cache
  - Updating doctrine/doctrine-bundle (1.10.2 => 1.11.1): Loading from cache
  - Updating doctrine/instantiator (1.1.0 => 1.2.0): Loading from cache
  - Updating nikic/php-parser (v4.2.1 => v4.2.2): Loading from cache
  - Updating twig/twig (v2.6.2 => v2.11.0): Loading from cache
  - Updating symfony/twig-bridge (v4.2.4 => v4.2.9): Loading from cache
  - Updating symfony/twig-bundle (v4.2.7 => v4.2.9): Loading from cache
  - Updating symfony/web-profiler-bundle (v4.2.8 => v4.2.9): Loading from cache
  - Updating phpdocumentor/reflection-docblock (4.3.0 => 4.3.1): Loading from cache
  - Updating doctrine/migrations (v2.0.0 => v2.0.2): Loading from cache
```

**How to reproduce**  
install latest 4.2.9 and update with dependencies. Create a controller with a FormType with some required value. Then write an automated test that simply saves the form empty.

```
$crawler = $client->request('GET', '/test');
$form = $crawler->selectButton('Save')->form();
```

It seems the problem is not when you use env=dev, because the html5 required will trigger and disallow the submit.. still looking into it

**Possible Solution**  
Probably dependency version of twig-bundle should be fixed or issue should be fixed in twig-bundle by having some kind of non-breaking fallback.

**Additional context**

**Comment by javiereguiluz** [comment]
This feature was added to Symfony 4.3 in #28635, so you are right that it shouldn't be used in deps installable via Symfony 4.2.

**Comment by xabbuh** [comment]
It looks like you somehow ended up using Twig bridge 4.3 with version 4.2 of the Form component. I don't see how that can happen as the Twig bridge 4.3 conflicts with any version of the Form component before 4.3.

**Comment by rbaarsma** [comment]
I did try to update to Symfony 4.3 earlier, but I got too many deprecations and decided to update latest 4.2.* instead. Just to be sure, I removed both `vendors` and `bin/.phpunit` directories and did a `composer install` again and re-tested, but still the same problem

**Comment by rbaarsma** [comment]
Accodring to the composer.lock, I am using the symfony/twig-bridge at version 4.2.9

```
            "name": "symfony/twig-bridge",
            "version": "v4.2.9",
```

**Comment by xabbuh** [comment]
Can you show the output of `composer show`?

**Comment by rbaarsma** [comment]
```
aws/aws-sdk-php                     3.89.0  AWS SDK for PHP - Use Amazon Web Services in your PHP project
beberlei/assert                     v3.2.0  Thin assertion library for input validation in business models.
bugsnag/bugsnag                     v3.16.0 Official Bugsnag notifier for PHP applications.
bugsnag/bugsnag-symfony             v1.5.0  Official Bugsnag notifier for Symfony applications.
clue/stream-filter                  v1.4.1  A simple and modern approach to stream filtering in PHP
composer/ca-bundle                  1.1.4   Lets you find a path to the system CA bundle, and includes a fallback to the Mozilla CA bundle.
composer/semver                     1.4.2   Semver library that offers utilities, version constraint parsing and validation.
composer/xdebug-handler             1.3.2   Restarts a process without xdebug.
csa/guzzle-bundle                   v3.1.1  A bundle integrating GuzzleHttp >= 4.0
csa/guzzle-cache-middleware         v1.0.0  A Cache middleware for GuzzleHttp >= 6.0
csa/guzzle-history-middleware       v1.0.0  A History middleware for GuzzleHttp >= 6.0
csa/guzzle-stopwatch-middleware     v1.0.0  A Stopwatch middleware for GuzzleHttp >= 6.0
dama/doctrine-test-bundle           v5.0.2  Symfony bundle to isolate doctrine database tests and improve test performance
defuse/php-encryption               v2.2.1  Secure PHP Encryption Library
doctrine/annotations                v1.6.1  Docblock Annotations Parser
doctrine/cache                      v1.8.0  Caching library offering an object-oriented API for many cache backends
doctrine/collections                v1.6.1  PHP Doctrine Collections library that adds additional functionality on top of PHP arrays.
doctrine/common                     v2.10.0 PHP Doctrine Common project is a library that provides additional functionality that other Doctr...
doctrine/data-fixtures              v1.3.1  Data Fixtures for all Doctrine Object Managers
doctrine/dbal                       v2.9.2  Powerful PHP database abstraction layer (DBAL) with many features for database schema introspect...
doctrine/doctrine-bundle            1.11.1  Symfony DoctrineBundle
doctrine/doctrine-cache-bundle      1.3.5   Symfony Bundle for Doctrine Cache
doctrine/doctrine-fixtures-bundle   3.1.0   Symfony DoctrineFixturesBundle
doctrine/doctrine-migrations-bundle v2.0.0  Symfony DoctrineMigrationsBundle
doctrine/event-manager              v1.0.0  Doctrine Event Manager component
doctrine/inflector                  v1.3.0  Common String Manipulations with regard to casing and singular/plural rules.
doctrine/instantiator               1.2.0   A small, lightweight utility to instantiate objects in PHP without invoking their constructors
doctrine/lexer                      v1.0.1  Base library for a lexer that can be used in Top-Down, Recursive Descent Parsers.
doctrine/migrations                 v2.0.2  PHP Doctrine Migrations project offer additional functionality on top of the database abstractio...
doctrine/orm                        v2.6.3  Object-Relational-Mapper for PHP
doctrine/persistence                1.1.1   The Doctrine Persistence project is a set of shared interfaces and functionality that the differ...
doctrine/reflection                 v1.0.0  Doctrine Reflection component
easycorp/easy-log-handler           v1.0.7  A handler for Monolog that optimizes log messages to be processed by humans instead of software....
elasticsearch/elasticsearch         v6.1.0  PHP Client for Elasticsearch
enqueue/dbal                        0.9.9   Message Queue Doctrine DBAL Transport
enqueue/dsn                         0.9.2   Parse DSN
enqueue/enqueue                     0.9.11  Message Queue Library
enqueue/enqueue-bundle              0.9.10  Message Queue Bundle
enqueue/null                        0.9.2   Enqueue Null transport
enqueue/sqs                         0.9.11  Message Queue Amazon SQS Transport
facebook/webdriver                  1.6.0   A PHP client for Selenium WebDriver
fig/link-util                       1.0.0   Common utility implementations for HTTP links
friendsofphp/php-cs-fixer           v2.14.2 A tool to automatically fix PHP code style
friendsofsymfony/elastica-bundle    v5.0.3  Elasticsearch PHP integration for your Symfony project using Elastica
friendsofsymfony/rest-bundle        2.5.0   This Bundle provides various tools to rapidly develop RESTful API's with Symfony
guzzlehttp/guzzle                   6.3.3   Guzzle is a PHP HTTP client library
guzzlehttp/promises                 v1.3.1  Guzzle promises library
guzzlehttp/psr7                     1.5.2   PSR-7 message implementation that also provides common utility methods
guzzlehttp/ringphp                  1.1.1   Provides a simple API and specification that abstracts away the details of HTTP into a single PH...
guzzlehttp/streams                  3.0.0   Provides a simple abstraction over streams of data
hamcrest/hamcrest-php               v2.0.0  This is the PHP port of Hamcrest Matchers
jdorn/sql-formatter                 v1.2.17 a PHP SQL highlighting library
jean85/pretty-package-versions      1.2     A wrapper for ocramius/package-versions to get pretty versions strings
jms/metadata                        1.7.0   Class/method/property metadata management in PHP
jms/parser-lib                      1.0.0   A library for easily creating recursive-descent parsers.
jms/serializer                      1.13.0  Library for (de-)serializing data of any complexity; supports XML, JSON, and YAML.
jms/serializer-bundle               2.4.3   Allows you to easily serialize, and deserialize data of any complexity
lcobucci/jwt                        3.2.5   A simple library to work with JSON Web Token and JSON Web Signature
league/event                        2.2.0   Event package
league/flysystem                    1.0.52  Filesystem abstraction: Many filesystems, one API.
league/flysystem-aws-s3-v3          1.0.22  Flysystem adapter for the AWS S3 SDK v3.x
league/oauth2-server                7.2.0   A lightweight and powerful OAuth 2.0 authorization and resource server library with support for ...
liuggio/fastest                     v1.6.1  Simple parallel testing execution... with some goodies for functional tests.
mockery/mockery                     1.2.2   Mockery is a simple yet flexible PHP mock object framework
monolog/monolog                     1.24.0  Sends your logs to files, sockets, inboxes, databases and various web services
mtdowling/jmespath.php              2.4.0   Declaratively specify how to extract elements from a JSON document
nelmio/cors-bundle                  1.5.5   Adds CORS (Cross-Origin Resource Sharing) headers support in your Symfony2 application
nette/bootstrap                     v2.4.6  🅱 Nette Bootstrap: the simple way to configure and bootstrap your Nette application.
nette/di                            v2.4.15 💎 Nette Dependency Injection Container: Flexible, compiled and full-featured DIC with perfect...
nette/finder                        v2.5.0  🔍 Nette Finder: find files and directories with an intuitive API.
nette/neon                          v3.0.0  🍸 Nette NEON: encodes and decodes NEON file format.
nette/php-generator                 v3.2.1  🐘 Nette PHP Generator: generates neat PHP code for you. Supports new PHP 7.3 features.
nette/robot-loader                  v3.1.1  🍀 Nette RobotLoader: high performance and comfortable autoloader that will search and autoloa...
nette/utils                         v2.5.3  🛠 Nette Utils: lightweight utilities for string & array manipulation, image handling, safe JS...
nexylan/slack                       v2.2.0  A simple PHP package for sending messages to Slack, with a focus on ease of use and elegant syntax.
nexylan/slack-bundle                v2.1.0  Symfony bundle integration of nexylan/slack library
nikic/php-parser                    v4.2.2  A PHP parser written in PHP
ocramius/package-versions           1.4.0   Composer plugin that provides efficient querying for installed package versions (no runtime IO)
ocramius/proxy-manager              2.2.2   A library providing utilities to generate, instantiate and generally operate with Object Proxies
oneup/flysystem-bundle              3.0.3   Integrates Flysystem filesystem abstraction library to your Symfony2 project.
pagerfanta/pagerfanta               v1.1.0  Pagination for PHP 5.3
php-cs-fixer/diff                   v1.3.0  sebastian/diff v2 backport support for PHP5.6
php-http/client-common              2.0.0   Common HTTP Client implementations and tools for HTTPlug
php-http/discovery                  1.6.1   Finds installed HTTPlug implementations and PSR-7 message factories
php-http/guzzle6-adapter            v2.0.1  Guzzle 6 HTTP Adapter
php-http/httplug                    v2.0.0  HTTPlug, the HTTP client abstraction for PHP
php-http/httplug-bundle             1.15.2  Symfony integration for HTTPlug
php-http/logger-plugin              1.1.0   PSR-3 Logger plugin for HTTPlug
php-http/message                    1.7.2   HTTP Message related tools
php-http/message-factory            v1.0.2  Factory interfaces for PSR-7 HTTP Message
php-http/mock-client                1.3.0   Mock HTTP client
php-http/promise                    v1.0.0  Promise used for asynchronous HTTP requests
php-http/stopwatch-plugin           1.2.0   Symfony Stopwatch plugin for HTTPlug
phpcollection/phpcollection         0.5.0   General-Purpose Collection Library for PHP
phpdocumentor/reflection-common     1.0.1   Common reflection classes used by phpdocumentor to reflect the code structure
phpdocumentor/reflection-docblock   4.3.1   With this component, a library can provide support for annotations via DocBlocks or otherwise re...
phpdocumentor/type-resolver         0.4.0  
phpoption/phpoption                 1.5.0   Option Type for PHP
phpstan/phpdoc-parser               0.3.1   PHPDoc parser with support for nullable, intersection and generic types
phpstan/phpstan                     0.10.8  PHPStan - PHP Static Analysis Tool
phpstan/phpstan-strict-rules        0.10.1  Extra strict and opinionated rules for PHPStan
predis/predis                       v1.1.1  Flexible and feature-complete Redis client for PHP and HHVM
psr/cache                           1.0.1   Common interface for caching libraries
psr/container                       1.0.0   Common Container Interface (PHP FIG PSR-11)
psr/http-client                     1.0.0   Common interface for HTTP clients
psr/http-message                    1.0.1   Common interface for HTTP messages
psr/link                            1.0.0   Common interfaces for HTTP links
psr/log                             1.1.0   Common interface for logging libraries
psr/simple-cache                    1.0.1   Common interfaces for simple caching
queue-interop/amqp-interop          0.8.1  
queue-interop/queue-interop         0.8.0   Promoting the interoperability of MQs objects. Based on Java JMS
ralouphie/getallheaders             2.0.5   A polyfill for getallheaders.
ramsey/uuid                         3.8.0   Formerly rhumsaa/uuid. A PHP 5.4+ library for generating RFC 4122 version 1, 3, 4, and 5 univers...
react/promise                       v2.7.1  A lightweight implementation of CommonJS Promises/A for PHP
ruflin/elastica                     6.1.1   Elasticsearch Client
sensio/framework-extra-bundle       v5.2.4  This bundle provides a way to configure your controllers with annotations
snc/redis-bundle                    2.1.9   A Redis bundle for Symfony
squizlabs/php_codesniffer           3.4.0   PHP_CodeSniffer tokenizes PHP, JavaScript and CSS files and detects violations of a defined set ...
symfony/asset                       v4.2.9  Symfony Asset Component
symfony/browser-kit                 v4.2.9  Symfony BrowserKit Component
symfony/cache                       v4.2.9  Symfony Cache component with PSR-6, PSR-16, and tags
symfony/config                      v4.2.9  Symfony Config Component
symfony/console                     v4.2.9  Symfony Console Component
symfony/contracts                   v1.1.2  A set of abstractions extracted out of the Symfony components
symfony/css-selector                v4.2.9  Symfony CssSelector Component
symfony/debug                       v4.2.9  Symfony Debug Component
symfony/debug-bundle                v4.2.9  Symfony DebugBundle
symfony/debug-pack                  v1.0.7  A debug pack for Symfony projects
symfony/dependency-injection        v4.2.9  Symfony DependencyInjection Component
symfony/doctrine-bridge             v4.2.9  Symfony Doctrine Bridge
symfony/dom-crawler                 v4.2.9  Symfony DomCrawler Component
symfony/dotenv                      v4.2.9  Registers environment variables from a .env file
symfony/event-dispatcher            v4.2.9  Symfony EventDispatcher Component
symfony/expression-language         v4.2.9  Symfony ExpressionLanguage Component
symfony/filesystem                  v4.2.9  Symfony Filesystem Component
symfony/finder                      v4.2.9  Symfony Finder Component
symfony/flex                        v1.2.5  Composer plugin for Symfony
symfony/form                        v4.2.9  Symfony Form Component
symfony/framework-bundle            v4.2.9  Symfony FrameworkBundle
symfony/http-foundation             v4.2.9  Symfony HttpFoundation Component
symfony/http-kernel                 v4.2.9  Symfony HttpKernel Component
symfony/inflector                   v4.2.9  Symfony Inflector Component
symfony/intl                        v4.2.9  A PHP replacement layer for the C intl extension that includes additional data from the ICU libr...
symfony/lock                        v4.2.9  Symfony Lock Component
symfony/maker-bundle                v1.11.6 Symfony Maker helps you create empty commands, controllers, form classes, tests and more so you ...
symfony/monolog-bridge              v4.2.9  Symfony Monolog Bridge
symfony/monolog-bundle              v3.3.1  Symfony MonologBundle
symfony/options-resolver            v4.2.9  Symfony OptionsResolver Component
symfony/orm-pack                    v1.0.6  A pack for the Doctrine ORM
symfony/panther                     v0.3.0  A browser testing and web scraping library for PHP and Symfony.
symfony/phpunit-bridge              v4.3.0  Symfony PHPUnit Bridge
symfony/polyfill-intl-icu           v1.11.0 Symfony polyfill for intl's ICU-related data and classes
symfony/polyfill-mbstring           v1.11.0 Symfony polyfill for the Mbstring extension
symfony/polyfill-php72              v1.11.0 Symfony polyfill backporting some PHP 7.2+ features to lower PHP versions
symfony/process                     v4.2.9  Symfony Process Component
symfony/profiler-pack               v1.0.4  A pack for the Symfony web profiler
symfony/property-access             v4.2.9  Symfony PropertyAccess Component
symfony/property-info               v4.2.9  Symfony Property Info Component
symfony/psr-http-message-bridge     v1.0.2  PSR HTTP message bridge
symfony/routing                     v4.2.9  Symfony Routing Component
symfony/security                    v4.2.9  Symfony Security Component
symfony/security-bundle             v4.2.9  Symfony SecurityBundle
symfony/serializer                  v4.2.9  Symfony Serializer Component
symfony/serializer-pack             v1.0.2  A pack for the Symfony serializer
symfony/stopwatch                   v4.2.9  Symfony Stopwatch Component
symfony/templating                  v4.2.9  Symfony Templating Component
symfony/test-pack                   v1.0.5  A pack for functional and end-to-end testing within a Symfony app
symfony/translation                 v4.2.9  Symfony Translation Component
symfony/twig-bridge                 v4.2.9  Symfony Twig Bridge
symfony/twig-bundle                 v4.2.9  Symfony TwigBundle
symfony/validator                   v4.2.9  Symfony Validator Component
symfony/var-dumper                  v4.2.9  Symfony mechanism for exploring and dumping PHP variables
symfony/var-exporter                v4.2.9  A blend of var_export() + serialize() to turn any serializable data structure to plain PHP code
symfony/web-link                    v4.2.9  Symfony WebLink Component
symfony/web-profiler-bundle         v4.2.9  Symfony WebProfilerBundle
symfony/web-server-bundle           v4.2.9  Symfony WebServerBundle
symfony/yaml                        v4.2.9  Symfony Yaml Component
trikoder/oauth2-bundle              v1.1.0  Symfony bundle which provides OAuth 2.0 authorization/resource server capabilities.
twig/twig                           v2.11.0 Twig, the flexible, fast, and secure template language for PHP
webmozart/assert                    1.4.0   Assertions to validate method input/output with nice error messages.
willdurand/jsonp-callback-validator v1.1.0  JSONP callback validator.
willdurand/negotiation              v2.3.1  Content Negotiation tools for PHP provided as a standalone library.
zendframework/zend-code             3.3.1   provides facilities to generate arbitrary code using an object oriented interface
zendframework/zend-diactoros        1.8.6   PSR HTTP Message implementations
zendframework/zend-eventmanager     3.2.1   Trigger and listen to events within a PHP application
zircote/swagger-php                 3.0.2   swagger-php - Generate interactive documentation for your RESTful API using phpdoc annotations
```

**Comment by xabbuh** [comment]
This looks good. Can you create a small example application that allows to reproduce the issue?

**Comment by rbaarsma** [comment]
yes, doing that right now. How should I get it to you?

**Comment by xabbuh** [comment]
Just create a GitHub repository and link to it here.

**Comment by rbaarsma** [comment]
I tried and tried. Spent a couple of hours on this, even molested the repository I am working on to it's bare minimum and when I at some point removed all files and removed a bunch of third party bundles, finally the problem just disappeared. I've reset the git branch and installed composer again, doing the same upgrade, but I am now unable to reproduce the same issue. Closing as a weird coincidence..
